src/actions/notificationPane.js

```diff
@@ -0,0 +1,116 @@
+import axios from 'axios';
+import io from 'socket.io-client';
+export const GET_NOTIFICATIONS = 'GET_NOTIFICATIONS';
+export const GET_SOCKET_NOTIFICATIONS = 'GET_SOCKET_NOTIFICATIONS';
+export const MARK_NOTIFICATIONAS_READ = 'MARK_NOTIFICATIONAS_READ';
+export const CHANGE_NOTIFICATION_PANE_HEIGHT =
+	'CHANGE_NOTIFICATION_PANE_HEIGHT';
+
+export const GetNotifications = () => async dispatch => {
+	const Token = `Bearer ${localStorage.getItem('token')}`;
+	const headers = {
+		'Content-Type': 'application/json',
+		token: Token,
+	};
+	await axios
+		.get(`${process.env.BACKEND_BASE_URL}/api/v1/notifications`, { headers })
+		.then(res => {
+			const UserNotifications = res.data.data;
+			dispatch({
+				type: GET_NOTIFICATIONS,
+				Notifications: UserNotifications,
+			});
+		});
+};
+export const notificationPaneHeigthHandle = (
```

Comment:
@shemaeric1234 does this piece of information need to be in the global state? The panel height can be managed by the local state of the component.